test/unit/resources/http_test.rb

```diff
@@ -243,4 +243,19 @@
       _(headers.D).must_equal(4)
     end
   end
+
+  describe "Windows" do 
+    let(:backend)     { MockLoader.new.backend }
```

Comment:
The problem I'm running into now is passing this `inspec` object as being on Windows so that `inspec.os.windows?` is true

Reply:
Would the aim to be running the *same* test for HTTP - but running that test on a windows machine and ensuring it passes? Usually I'd run these tests actually from inside a windows machine to ensure they're working.

Reply:
From what I'm seeing it can be done in 2 ways
1) use the mock helper to match the generated command to a response
2) run this on windows (though it won't work yet as the local VS remote uses different classes. 
For now, I'd like to get remote working and then add local after.

Reply:
To get `inspec` to mock as windows, I modified line 248 to read:
```
    let(:backend)     { MockLoader.new(:windows).backend }
```
You'll also need to stub the detection of WebRequest, using something like this in the `it` block:
```
Inspec::Resources::Cmd.any_instance
          .stubs(:exist?)
          .returns(true)
```
After doing those two things, I still encountered a few minor errors regarding the assemble of the request_header_string, but those were easily corrected.